torch/testing/_internal/distributed/rpc/jit/rpc_test.py

```diff
@@ -1,59 +1,63 @@
-from typing import Dict, Tuple
+import time
+from typing import Dict, List, Tuple
 
 import torch
-import time
 import torch.distributed as dist
 import torch.distributed.rpc as rpc
-from torch.distributed.rpc.internal import _build_rpc_profiling_key, RPCExecMode
 from torch import Tensor
 from torch.distributed.rpc import RRef
+from torch.distributed.rpc.internal import RPCExecMode, _build_rpc_profiling_key
+from torch.futures import Future
 from torch.testing._internal.common_utils import TemporaryFileName
 from torch.testing._internal.dist_utils import (
     dist_init,
+    get_function_event,
     initialize_pg,
     worker_name,
-    get_function_event
 )
 from torch.testing._internal.distributed.rpc.rpc_agent_test_fixture import (
     RpcAgentTestFixture,
 )
 
+
 def sleep(t):
     time.sleep(t)
 
+
 def rpc_return_rref(dst):
     return rpc.remote(dst, torch.add, args=(torch.ones(2, 2), 1))
 
+
 @torch.jit.script
 def rref_local_value(rref: RRef[Tensor]) -> Tensor:
     return rref.local_value()
 
+
 @torch.jit.script
-def list_create():
-    # type: () -> List[int]
+def list_create() -> List[int]:
     global_list = [1, 2, 3]
     return global_list
 
+
 @torch.jit.script
-def rref_list_mutate(rref):
-    # type: (RRef[List[int]]) -> None
+def rref_list_mutate(rref: RRef[List[int]]) -> None:
     rref.local_value().append(4)
     rref.to_here().append(5)
     rref.to_here(5.0).append(6)
 
-def return_value(value):
-    # type: (int) -> int
```

Comment:
Is this deprecated? If so, we will also need to modify docs

Reply:
Yes, python2 type comment should be deprecated, since we only support Python3 now.